### ISSUE TITLE
Error if vite.config.js not present

### ISSUE THREAD

**Issue by ALMaclaine** [pzmosquito/eslint-import-resolver-vite#3]
My component library doesn't have a vite.config.js in root because it's not needed.

Took me a while to figure out that's why the lib didn't work, I had to setup the debugger.

Please consider erroring if vite.config.js not present in root directory.

I'm willing to put in a PR if you would like.

**Comment by pzmosquito** [comment]
You can use [configPath](https://github.com/pzmosquito/eslint-import-resolver-vite#config-options) to set the path of the `vite.config.js` file.

```js
"import/resolver": {
    vite: {
        configPath: "./app1/vite.confg.js"
    }
}
```

**Comment by ALMaclaine** [comment]
yes, that's not what im asking, im asking for an error when the default is missing so that i dont have to run debug to see that im missing my config.

**Comment by pzmosquito** [comment]
I can definitely put in some checks, but this plugin is for resolving the alias in the vite config file, if you don't have a config file, then you don't even need this plugin. Can you tell me why you want to install this plugin without using alias?

**Comment by ALMaclaine** [comment]
Because my component library didn't need a vite.config.js, it's just a set of components that are used from other repos which have their own vite config.

I added eslint-config-import and was having problems because my resolver wasn't working because my project is set up to satisfy vite (most testing was done from storybook which has it's own vite builder).

So upon adding eslint-config-import and eslint-import-resolver-vite I had failures, and it wasn't immediately obviously why. I had to get the debugger setup and read through the errors. While I admit this maybe an uncommon case, it would have saved me a few hours if I got an error that the config that was being looked for doesn't exist.

In reference to the alias comment, the plugin has value for no-unresolved rule even when not using aliases.
